### DISCUSSION TITLE
Rain wallpaper Open Controls topright

### DISCUSSION THREAD

**Discussion by ritul-patel** [rocksdanister/lively#2282] [General]
my desktop home page show open control, how remove ,he is IrritatingMy desktop homepage displays an "open control" that is irritating. How can I remove it?
![Uploading Screenshot 2024-06-16 165136.png…]()

**Comment by ritul-patel** [comment]
[
![Screenshot 2024-06-16 165136](https://github.com/rocksdanister/lively/assets/143929587/9d33f102-60ba-41a1-a7c9-b4ef083380a3)
](url)

**Comment by rocksdanister** [comment]
Uncheck `debug`

<img width="308" alt="image" src="https://github.com/rocksdanister/lively/assets/17554161/c9cb509c-0878-41e7-a4b5-c07aabc33183">

**Reply by ritul-patel** [reply]
thank you
